ports/rapidjson/portfile.cmake

```
@@ -2,14 +2,14 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Tencent/rapidjson
-    REF ce81bc9edfe773667a7a4454ba81dac72ed4364c # accessed on 2020-09-14
-    SHA512 a1bbc5668fca9ee34b05ae424a5177b66beaf7b3b4cdfbb3cb067fc58ce91717b4030e2c268ad32e4aa4a7ef59ab5e3aae05ff71d0d91d463eafa7d2e077df07
+    REF 0d4517f15a8d7167ba9ae67f3f22a559ca841e3b # accessed on 2021-11-01
```

```suggestion
    REF 232389d4f1012dddec4ef84861face2d2ba85709 # accessed on 2022-6-28
````
Refer to the update frequency of the upstream release version, we should update to the latest master commit.